app/client/cli/servicer.go

```diff
@@ -0,0 +1,284 @@
+package cli
+
+import (
+	"context"
+	sha "crypto"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/pocket/rpc"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/crypto"
+)
+
+func init() {
+	rootCmd.AddCommand(NewServicerCommand())
+}
+
+// TECHDEBT: (unittest) unit test the command: e.g. on number of arguments
+func NewServicerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Servicer",
+		Short:   "Servicer specific commands",
+		Aliases: []string{"servicer"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmds := servicerCommands()
+	applySubcommandOptions(cmds, attachPwdFlagToSubcommands())
+	applySubcommandOptions(cmds, attachKeybaseFlagsToSubcommands())
+
+	cmd.AddCommand(cmds...)
+
+	return cmd
+}
+
+func servicerCommands() []*cobra.Command {
+	cmdDef := actorCmdDef{"Servicer", coreTypes.ActorType_ACTOR_TYPE_SERVICER}
+	cmds := []*cobra.Command{
+		newStakeCmd(cmdDef),
+		newEditStakeCmd(cmdDef),
+		newUnstakeCmd(cmdDef),
+		newUnpauseCmd(cmdDef),
+		{
+			Use:   "Relay <servicer> <application> <relayChainID> <payload>",
+			Short: "Relay <servicer> <application> <relayChainID> <payload>",
+			Long: `Sends a trustless relay using <payload> as contents, to the specified active <servicer> in the the <application>'s session.
+Will prompt the user for the *application* account passphrase`,
+			Aliases: []string{},
+			Args:    cobra.ExactArgs(4),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				servicerAddr := args[0]
+				applicationAddr := args[1]
+				chain := args[2]
+				relayPayload := args[3]
+
+				// TODO: (SUGGESTION) refactor to decouple the client logic from the CLI/command
+				pk, err := getPrivateKey(applicationAddr)
+				if err != nil {
+					return fmt.Errorf("error getting application's private key: %w", err)
+				}
+
+				session, servicer, err := fetchServicer(cmd.Context(), applicationAddr, chain, servicerAddr)
```

Comment:
Thoughts on decoupling "getSession" and "getServicer" or getting the session in place and passing that to `fetchServicer`?

Feels weird to get back a session from a function called `fetchServicer`

Reply:
Thank you for catching this. Yes, the `fetchServicer` signature looked strange. Fixed and it reads a lot better now IMO.